src/ralph/deployment/views.py

```diff
@@ -0,0 +1,141 @@
+import logging
+
+from django.conf import settings
+from django.core.exceptions import SuspiciousOperation
+from django.core.urlresolvers import reverse
+from django.http import Http404, HttpResponse, HttpResponseRedirect
+from django.template import Context, Template
+
+from ralph.admin.helpers import get_client_ip
+from ralph.deployment.models import Deployment
+
+logger = logging.getLogger(__name__)
+
+# TODO: find better solution
+RALPH_INSTANCE = getattr(settings, 'RALPH_INSTANCE', 'http://127.0.0.1:8000')
+
+
+def _get_preboot(deployment_id):
+    preboot = None
+    try:
+        preboot = Deployment.objects.get(id=deployment_id).preboot
+    except ValueError:
+        logger.warning('Incorrect UUID: {}'.format(deployment_id))
+        raise SuspiciousOperation('Malformed UUID')
+    except Deployment.DoesNotExist:
+        logger.warning('Deployment with UUID: {} doesn\'t exist' .format(
+            deployment_id
+        ))
+        raise Http404
+    return preboot
+
+
+def _render_configuration(configuration, deployment):
+    template = Template(configuration)
+    context = Context({
+        'ralph_instance': RALPH_INSTANCE,
+        'deployment_id': deployment.id,
+        'kickstart': RALPH_INSTANCE + reverse('deployment_kickstart', kwargs={
```

Comment:
We could but now you can add simple ``{{ kickstart }}`` to template instead of ``{{ ralph_instance }}{{ kickstart }}``. IMHO if user insert one variable to template then probably do it correctly.